src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <header class="search-container">
+      <form
+        class="search-bar top-search-bar"
+        data-qa="big"
+      >
+        <input
+          class="search-bar-input search-bar-input--top"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        class="search-bar bottom-search-bar"
+        data-qa="small"
+      >
+        <input
+          class="search-bar-input search-bar-input--bottom"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
The attributes in the `input` tag should start on a new line with 2-space indentation relative to the tag, and the closing bracket should be on the same level as the opening one. Please adjust the formatting of the `input` tag to match the guideline.